libs/ui/elements/src/lib/metadata-contact/metadata-contact.component.html

```diff
@@ -19,6 +19,7 @@
           class="text-primary font-title text-21 mr-2 cursor-pointer hover:underline"
           (click)="onOrganizationClick()"
           data-cy="organization-name"
+          *ngIf="shownOrganization?.name"
```

Comment:
was that change intentional? I'm not sure effect it has but it looks not intended :wink:

Reply:
It was intentional! I had a case where the organization had no name (or an error on it perhaps), so I added it as a fallback. I know it's not included in the PR, but figured it was shorter a fix than creating a ticket for it...

Reply:
ok, then I would suggest trying to identify what would cause an organization to have no name. This can have other implications so I think it'd be best if you could go ahead and create a bug ticket? thanks!